iphone/Maps/PrivacyInfo.xcprivacy

```diff
@@ -23,6 +23,7 @@
 		<dict>
 			<key>NSPrivacyAccessedAPITypeReasons</key>
 			<array>
+				<string>C617.1</string>
```

Comment:
This should go to `AccessedAPICategoryFileTimestamp`, right?

Reply:
Thanks! This code is set incorrectly here. I'll fix it. This is for the timestamp